.github/workflows/python-lint.yaml

```diff
@@ -0,0 +1,21 @@
+name: Python lint
+
+on:
+  push:
+    paths:
+      - deployer/
+
+jobs:
+  flake8_py3:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup Python
+        uses: actions/setup-python@v1
```

Comment:
Is there any reason for not using the v2 version of the action? (I don't know how they differ in terms of features tho...)

Reply:
Ah good catch! I've updated :)